next.config.js

```diff
@@ -36,6 +37,7 @@ module.exports = withTypescript({
 
     config.resolve.alias['react-dom$'] = 'react-dom/profiling';
     config.resolve.alias['scheduler/tracing'] = 'scheduler/tracing-profiling';
+    config.resolve.alias = { ...config.resolve.alias, ...defaultAlias };
```

Comment:
The Babel plugin should be enough. The whole point is not to have to tell anything to webpack.